pom.xml

```diff
@@ -53,7 +53,7 @@
 		<!-- other third party libraries versions -->
 		<slf4j.version>1.7.7</slf4j.version>
 		<logback.version>1.0.13</logback.version>
-		<sesame.version>2.7.13</sesame.version>
+		<sesame.version>2.8.1</sesame.version>
```

Comment:
this shall be removed and reuse it from https://github.com/UnifiedViews/Core/blob/feature/OpenRdf_2.8.1/pom.xml#L56

Necessary to update version of parent pom to allign it,
It is also relevant for other versions that shall be common in API / Plugins / Helpers / Core because they are provided.